test/over_react/component_declaration/redux_component_test/test_reducer.g.dart

```diff
@@ -0,0 +1,222 @@
+// GENERATED CODE - DO NOT MODIFY BY HAND
```

Comment:
@jacehensley-wf is it necessary to commit generated files?

Reply:
If you don't commit, build it into your build process to build before tests run.

Reply:
Yeah, I can just build it during CI, but while deving you'll have analysis warnings till you generate